styles/Events.module.scss

```diff
@@ -44,6 +44,15 @@
   grid-template-rows: 1fr 1fr 1rem;
   grid-template-columns: 200px 1fr;
   text-decoration: none;
+  transition: .3s ease;
+  &:hover{
+    color: $white;
+    background: $black;
+    box-shadow: inset 0 0 20px;
```

Comment:
If you don't add any color, box-shadow will inherit the "color" property's value, what is white currently, in this case the background is white, so the box-shadow is not visible, is the box shadow really needed?, in that case, what color should be?

here's an example of how box-shadow works:

![lastrecord](https://user-images.githubusercontent.com/66505715/188671890-1bc70c99-e2fe-43f0-9c6b-8852b70415ff.gif)

CC: @angelik0828